build/entrypoint.sh

```diff
@@ -4,12 +4,19 @@ set -e
 
 : ${GCLOUD_REGISTRY:=gcr.io}
 : ${IMAGE:=$GITHUB_REPOSITORY}
+: ${ARGS:=} # Default: empty build args
 : ${TAG:=$GITHUB_SHA}
 : ${DEFAULT_BRANCH_TAG:=true}
+: ${LATEST:=false}
```

Comment:
this changes the default behaviour. could you leave LATEST by default, and disable it in your own usage?

Reply:
Thanks. I'll change the condition instead. By default, it will always have the `latest` tag.

Anyway, having `latest` tag violates the Docker's best practices.